server/@types/dto/index.d.ts

```diff
@@ -42,26 +44,28 @@ declare module 'dto' {
     goalReference: string
     prisonNumber: string
   }
-}
 
-declare module 'inductionDto' {
```

Comment:
This diff is quite hard to read; but essentially I have:

* Moved EducationDto out of the inductioDto module into the dto module (because its not related to inductions)
* Changed/corrected the property definition of EducationDto (mainly it's property qualifications)
* Moved the type ReferencedAndAuditable out of the inductionDto module to outside of any module, so that other dto's (like EducationDto) can use it